.github/ISSUE_TEMPLATE/1-roadmap-item.yml

```diff
@@ -0,0 +1,30 @@
+name: '📦 New Roadmap Item'
+labels: 'enhancement'
+description: >
+  This is a project which normally spans within a single across a few minor releases,
+  and that become major deliverables when shipped.
+  Potential examples are "Support for Java 22", "Problems HTML Summary Report", 
+  "Isolated Projects: Incubating Status" or "Declarative Gradle EAP 2".
+  Consider it "EPIC" in normal Agile terms.
+
+body:
+  - type: textarea
+    attributes:
+      label: Summary / Manifesto
+      description: > 
+        Describe what value this project brings to Gradle, and what areas/topics it should address.
```

Comment:
🤔 I think we should point out that the context should be set up:

(from the RN guidelines):
> Set the context - what are we even talking about (don’t assume too much about what the user already knows)